swan-lake/learn/using-the-cli-tools.md

```diff
@@ -126,27 +126,27 @@ These everyday commands are your best friends! They address the very basics of p
 <table class="cComandTable">
 <tr>
 <td class="cCommand">build</td>
-<td class="cDescription">Compile a Ballerina program, a single BAL file, a module, an entire project, or a single root module into an executable JAR file.
+<td class="cDescription">Compile a Ballerina program, a single BAL file, a module, an entire package, or a single root module into an executable JAR file. For more information, see <a href="/swan-lake/learn/running-ballerina-code">Running Ballerina Code</a>.
 </td>
 </tr>
 <tr>
 <td class="cCommand">run</td>
-<td class="cDescription">Build and run a Ballerina program, a single BAL file, an entire project, or a previously-built program. For more information, see <a href="/swan-lake/learn/running-ballerina-code">Running Ballerina Code</a>.
+<td class="cDescription">Build and run a Ballerina program, a single BAL file, an entire package, or a previously-built program. For more information, see <a href="/swan-lake/learn/running-ballerina-code">Running Ballerina Code</a>.
 </td>
 </tr>
 <tr>
 <td class="cCommand">test</td>
-<td class="cDescription">Run tests of a particular module or all the modules of a Ballerina project. For more information, see <a href="/swan-lake/learn/testing-ballerina-code/testing-quick-start/">Testing Ballerina Code</a>.
+<td class="cDescription">Run tests of a particular module or all the modules of a Ballerina package. For more information, see <a href="/swan-lake/learn/testing-ballerina-code/testing-quick-start/">Testing Ballerina Code</a>.
 </td>
 </tr>
 <tr>
 <td class="cCommand">doc</td>
-<td class="cDescription">Generate API documents for all public symbols of a Ballerina module or project. For more information, see <a href="/swan-lake/learn/documenting-ballerina-code">Documenting Ballerina Code</a>.
+<td class="cDescription">Generate API documents for all public symbols of a Ballerina module or package. For more information, see <a href="/swan-lake/learn/documenting-ballerina-code">Documenting Ballerina Code</a>.
```

Comment:
Only for package

Reply:
@rdulmina the output of the `bal help doc` is as following: 
![image](https://user-images.githubusercontent.com/9464997/105646716-763acd00-5ec7-11eb-8cf0-5b5c79056e75.png)
According to the provided description, the `bal doc` is used to build the documentation for a Ballerina package.

Reply:
Yes, then we should change `Generate API documents for all public symbols of a Ballerina module or package` to `Generate API documents for all public symbols of a Ballerina package`.

Reply:
Thanks, updated.